src/main/java/edu/ucsb/cs156/dining/controllers/ReviewController.java

```diff
@@ -221,25 +207,10 @@ public Review moderateReview(@Parameter Long id, @Parameter ModerationStatus sta
     }
 
     @Operation(summary = "See reviews that need moderation")
-    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_MODERATOR')")
     @GetMapping("/needsmoderation")
     public Iterable<Review> needsmoderation() {
         Iterable<Review> reviewsList = reviewRepository.findByStatus(ModerationStatus.AWAITING_REVIEW);
         return reviewsList;
     }
-    @Operation(summary = "Get a specific single review ID")
-    @PreAuthorize("hasRole('ROLE_USER')")
-    @GetMapping("/get")
-    public Review getReviewByID(@Parameter Long id) {
-        Review review = reviewRepository.findById(id).orElseThrow(
-                () -> new EntityNotFoundException(Review.class, id)
-        );
-
-        User current = getCurrentUser().getUser();
-        if(current.getId() != review.getReviewer().getId() && !current.getAdmin()) {
-            throw new AccessDeniedException("Only user who made this review or admin can get id");
-        }
-        return review;
-    }
-
-}
+}
```

Comment:
Why did you delete the "get by id" backend endpoint?  Was there a group decision by the whole team that this is no longer needed? Or are you accidentally removing someone else's code?